lib/livebook_web/live/home_live/edit_sessions_component.ex

```diff
@@ -0,0 +1,87 @@
+defmodule LivebookWeb.HomeLive.EditSessionsComponent do
+  use LivebookWeb, :live_component
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div class="p-6 pb-4 flex flex-col space-y-8">
+      <h3 class="text-2xl font-semibold text-gray-800">
+        <%= title(@action) %>
+      </h3>
+      <.message action={@action} selected_session_ids={@selected_session_ids} sessions={@sessions}/>
+      <div class="mt-8 flex justify-end space-x-2">
+        <button class="button-base button-red" phx-click={@action} phx-target={@myself}>
+          <.remix_icon icon="close-circle-line" class="align-middle mr-1" />
+          <%= button_label(@action) %>
+        </button>
+        <%= live_patch "Cancel", to: @return_to, class: "button-base button-outlined-gray" %>
+      </div>
+    </div>
+    """
+  end
+
+  defp message(%{action: "close_all"} = assigns) do
+    ~H"""
+    <p class="text-gray-700">
+      Are you sure you want to close <%= maybe_pluralize(length(@selected_session_ids), "session") %>?
+      <%= if not_persisted(assigns) == 1 do %>
```

Comment:
In the template we should only use individual assigns, so like `not_persisted(@sessions, @selected_session_ids)`.

(but this will be simpler anyway with the other suggestion :))